lib/Common/Memory/RecyclerPageAllocator.h

```diff
@@ -40,5 +40,6 @@ class RecyclerPageAllocator : public IdleDecommitPageAllocator
     Recycler* recycler;
 
     bool IsMemProtectMode();
+    virtual char * TryAllocFromZeroPages(uint pageCount, PageSegmentBase<VirtualAllocWrapper> ** pageSegment, PageHeapMode pageHeapFlags) override;
```

Comment:
Can we avoid the virtual here? 
I think you can just check for HasZeroPageQueue()?